EligibilityCheckActivity/index.ts

```diff
@@ -1,166 +1,13 @@
-﻿import { AzureFunction, Context } from "@azure/functions";
-
-import { fromEither, tryCatch } from "fp-ts/lib/TaskEither";
-import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
-import * as t from "io-ts";
-import { readableReport } from "italia-ts-commons/lib/reporters";
-import { FiscalCode, NonEmptyString } from "italia-ts-commons/lib/strings";
-import * as path from "path";
-import * as soap from "soap";
-import { ConsultazioneSogliaIndicatore } from "../generated/SvcConsultazione/ConsultazioneSogliaIndicatore";
-import { ConsultazioneSogliaIndicatoreResponse_element_tns } from "../generated/SvcConsultazione/ConsultazioneSogliaIndicatoreResponse_element_tns";
-import {
-  SottoSogliaEnum,
-  TipoIndicatoreEnum
-} from "../generated/SvcConsultazione/ConsultazioneSogliaIndicatoreResponseType_type_tns";
-import { createClient, promisifySoapMethod, SoapMethodCB } from "../utils/soap";
+﻿import { getRequiredStringEnv } from "io-functions-commons/dist/src/utils/env";
+import { createClient } from "../utils/inpsSoapClient";
+import { getEligibilityCheckActivityHandler } from "./handler";
 
 const INPS_SERVICE_HOST = getRequiredStringEnv("INPS_SERVICE_HOST");
+const INPS_SERVICE_ENDPOINT = getRequiredStringEnv("INPS_SERVICE_ENDPOINT");
```

Comment:
Can we add these variable in the readme?

Reply:
fixed in 0c41302

Reply:
can we just use one variable instead of 4 and parse the url when needed ? ie:

`INPS_SERVICE_ENDPOINT=http://inps.it:50777/my/service/endpoint`